docker/runtime/ruby-2.4/event-trigger/kubeless.rb

```diff
@@ -45,7 +48,13 @@
 trap("TERM") { consumer.stop }
 
 consumer.each_message do |message|
-  mod.send(FUNC_HANDLER.to_sym, message.value)
+  begin
+    status = Timeout::timeout(ftimeout) {
+      mod.send(FUNC_HANDLER.to_sym, message.value)
+    }
+  rescue => e
```

Comment:
I think we should only rescue the Timeout::Error here

Reply:
This was just to avoid the pod to be restarted because whenever an unhandled exception appears but yes, we may want to rescue just timeout errors

Reply:
Ah okay, I'm not sure what makes sense so will leave it to your discretion. What do the other runtimes in Kubeless do, what do other serverless platforms do, etc.?

Reply:
actually in both python and nodejs we are catching the errors so we should do the same here to maintain the behavior